### ISSUE TITLE
Setting up DOTA for training and evaluation

### ISSUE THREAD

**Issue by Arka161** [yijingru/BBAVectors-Oriented-Object-Detection#97]
I looked into the DOTA website: 

```
https://captain-whu.github.io/DOTA/dataset.html
```

I see Training , evaluation and testing data. How should I paste these data into the `datasets` folder for testing BBA? Should I run the DOTA-Devkit too? 

I am confused on the set up steps. 

Thank you

**Comment by yijingru** [comment]
https://github.com/yijingru/BBAVectors-Oriented-Object-Detection#about-dota may help

**Comment by Arka161** [comment]
@yijingru Thank you for your reply. 

I currently have `datasets/images` and `datasets/LabelTxt` in my directory already. To obtain `trainval.txt` and `test.txt`, for running the code, could you suggest how to do it? Thanks

**Comment by Arka161** [comment]
I see you mention they are the file names, but I am having trouble understanding this block: 

```
P0000__0.5__0___0
P0000__0.5__0___1000
P0000__0.5__0___1500
P0000__0.5__0___2000
P0000__0.5__0___2151
P0000__0.5__0___500
P0000__0.5__1000___0
```

Thank you

**Comment by yijingru** [comment]
It is image split label. check this one https://github.com/CAPTAIN-WHU/DOTA_devkit#usage

**Comment by Arka161** [comment]
@yijingru Thank you for all the help. 

I managed to set the code base up for testing DOTA! While my own GPU is weak to train from scratch, I am able to perform Inference using your pre-trained model_50.pth file. 

There are some small bugs in the readme file, related to running the command for training. For example, the code has `num_epochs` instead of epochs, and it is not 100% clear what the train_labels and test_labels are, and that we have to do a train-test split using selection. 

Can I edit the README.md and make the instructions a little more clear for other researchers? Thanks. 

Either way, I think I am ready to close this issue.

**Comment by yijingru** [comment]
Of course you can. You can send pull request and I will review and merge. Thanks for your contribution!
